app/modules/node.c

```diff
@@ -604,6 +604,22 @@ static int node_egc_setmode(lua_State* L) {
   legc_set_mode( L, mode, limit );
   return 0;
 }
+//
+// Lua: osprint(true/false)
+// Allows you to turn on the native Espressif SDK printing
+static int node_osprint( lua_State* L )
+{
+  if (lua_toboolean(L, 1)) {
+    system_set_os_print(1);
+    os_install_putc1((void *)uart0_write_char);
```

Comment:
Do we really need to switch around the putc1 handlers? `ets_printf()` and co happily honour `system_set_os_print()`. I'm not keen on suddenly using different pins, especially when those might be wired up to something completely unrelated...